server.js

```diff
@@ -183,15 +200,30 @@ function handleSubscribeEvent (req, res) {
     });
 }
 
+
 /**
  * Get the topic name for a given item
  * @method getTopicFor
  * @param  {String}    device   Device Name
  * @param  {String}    property Property
+ * @param  {String}    type     Type of topic (command or state)
  * @return {String}             MQTT Topic name
  */
-function getTopicFor (device, property) {
-    return [config.mqtt.preface, device, property].join('/');
+function getTopicFor (device, property, type) {
+    var tree = [config.mqtt.preface, device, property],
+        suffix;
+
+    if (type === TOPIC_COMMAND) {
+        suffix = config.mqtt[SUFFIX_COMMAND];
+    } else if (type === TOPIC_STATE) {
+        suffix = config.mqtt[SUFFIX_STATE];
+    }
+
+    if (suffix) {
+        tree.push(suffix);
+    }
+
+    return tree.join('/');
```

Comment:
Looks like the default suffixes are empty strings. This line seems to add a trailing slash if you use the default suffixes. Is that the case and If so, will this be a problem for the topic's path?